local-setup/README.md

```diff
@@ -0,0 +1,37 @@
+#  How to use the local-setup
+
+## Prerequisite
+- worker built with `make`
+- integritee-node built with `cargo build --release --features skip-ias-check`
+(in case you have a valid intel key, you can omit the `--features skip-ias-check`, but you must not use the subcommand `--skip-ra` in the config.json for the worker startup).
+
```

Comment:
I think we need to be more precise:

You can only use a node without `skip-ias-check` iff:
* you have sgx hardware and compile the worker with `SGX_MODE=HW`, which is the default mode
* you have a valid intel IAS key (development key is fine).


PS. it is subcommand **flag** not just subcommand.

Reply:
Adapted accordingly.